frontend/src/components/pages/chat/components/messageList/MessageList.tsx

```diff
@@ -1,52 +1,87 @@
 import Box from '@material-ui/core/Box';
+import { DTO } from 'dto';
 import React, { useEffect, useRef, useState } from 'react';
 import { useParams } from 'react-router-dom';
 import { getChatMessages } from '../../../../../services/apiChatService';
+import { getUserDetails } from '../../../../../services/apiUserService';
+import { getUserFromStorage } from '../../../../../services/sessionStorageService';
 import { Compose } from '../compose/Compose';
 import { Message } from '../message/Message';
 import './messageList.scss';
 
-const MY_USER_ID = 'apple';
-
 export const MessageList: React.FC = () => {
-  const [messages, setMessages] = useState<Array<any>>([]);
+  const [messages, setMessages] = useState<Array<DTO.IMessage>>([]);
+  const [userId, setUserId] = useState('');
+  const [userName, setUserName] = useState('');
   const { dialogId } = useParams<any>();
+  const chatUserId = getUserFromStorage()._id;
```

Comment:
it is better to use same style for variables returned by function, compare lines 16 and 17